### DISCUSSION TITLE
Why is the top/bottom padding not working on buttons?

### DISCUSSION THREAD

**Discussion by jprateragg** [fomantic/Fomantic-UI#2030] [Elements (button, image, label, loader, ...)] [answered]
I'm using Fomantic-UI 2.8.6. Inside a table cell, I have several buttons. But, when the buttons spill over to the next line, the tops/bottoms are touching. Ex:

![SNAG-0000](https://user-images.githubusercontent.com/3075374/127037464-4abd1892-cc37-4f53-aadd-d23695ed9245.png)

However, if you look at the documentation for buttons (https://fomantic-ui.com/elements/button.html#basic), it shows the buttons with normal padding between new lines.

This is my HTML...I don't believe I have anything here that would prevent this from looking like what is in the documentation.

```
<tr id="tr_loan_action_buttons">
	<td colspan="2" class="center aligned" id="td_loan_action_buttons">
		
		<!-- REASSIGN APPROVER BUTTON -->
		
			<a href="xxx" class="ui blue button" title="Reassign" imodal="height=600;size=large"><i class="user icon"></i>Reassign</a>

		<!-- PREAPPROVAL CONDITIONS BUTTON -->
		
			<a href="xxx" class="ui teal button" title="Preapproval Conditions" imodal="height=500;size=large"><i class="list icon"></i>Preapproval Conditions</a>
		
		<!-- RETURN TO LOAN OFFICER BUTTON -->
		
			<a href="xxx" class="ui orange button" title="Return to LO" imodal="height=530"><i class="reply icon"></i>Return to LO</a>

		<!-- APPROVE BUTTON -->
		
			<a href="xxx" class="ui green disabled button" title="Approve Loan" imodal="height=550;size=large"><i class="check circle icon"></i>Approve</a>

		<!-- RESET ANALYSIS STATE BACK TO OPEN BUTTON -->
		
			<div class="ui purple button" id="reset_analysis_status" title="Reset Analysis Status"><i class="history icon"></i>Reset Analysis Status</div>

	</td>
</tr>
```
I am using vanilla Fomantic-UI. I have not customized it at all, done any custom build, etc.

**Comment by lubber-de** [comment, marked as answer]
This was already asked and answered here https://github.com/fomantic/Fomantic-UI/issues/2016#issuecomment-879297653
The docs have a custom CSS setting giving additional margin.

**Reply by jprateragg** [reply]
You are correct. That fixed it.Thanks!
